### PR TITLE
Updated README.md, Added CODE_OF_CONDUCT and Contributing.md

### PR DESCRIPTION
**Pull request by Sowham-3098**
Fixed issue: #49 
Added CODE_OF_CONDUCT.md and Contributing.md
Updated README.md with Contribution guideline

### CODE REVIEW

**Comment by Sowham-3098** [comment]
@AADI-1331 Check this

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+github@piyushacharya.com.
```
Comment (AADI-1331) [inline CODE_OF_CONDUCT.md]: whose he

**Comment by AADI-1331** [comment]
can u give the ss/video that how it is looking.

**Comment by Sowham-3098** [comment]
Screenshot (185)
Screenshot (186)

**Comment by Sowham-3098** [comment]
Screenshot (187)

**Comment by AADI-1331** [comment]
thanks for contributing Sowham, your name has been added in contributors list in readme.

**Pull request merged**